ghedt/peak_load_analysis_tool/ground_loads.py

```diff
@@ -305,11 +305,11 @@ def split_loads_by_month(self) -> None:
             # Day of month the peak heating load occurs
             # day of the month on which peak clg load occurs (e.g. 1-31)
             self.monthly_peak_cl_day[i] = \
-                math.floor(month_rejection_loads.index(
+                math.ceil(month_rejection_loads.index(
                     self.monthly_peak_cl[i]) / hours_in_day)
             # day of the month on which peak clg load occurs (e.g. 1-31)
             self.monthly_peak_hl_day[i] = \
-                math.floor(month_extraction_loads.index(
+                math.ceil(month_extraction_loads.index(
```

Comment:
Floor was the wrong option here. For example, if a peak load occurs on hour 7 in the month, then the day found is `7 / 24 = 0.29`. This should be considered to occur on the first day, not the 0th day, given that all things in `ground_loads` are 1 indexed.